graphein/ml/visualisation.py

```diff
@@ -115,13 +116,17 @@ def plot_pyg_data(
         d["coords"] = x.coords[i]
         if node_colour_tensor is not None:
             d["colour"] = float(node_colour_tensor[i])
+        if hasattr(x, "c_beta_vector"):
```

Comment:
This is a really nice idea. I think it would be better to generalise this and provide these as default arguments.

E.g.:

```python
for node_vec in node_vector_features:
    if hasattr(x, node_vec):
        d[node_feat] = getattr(x, node_feat)[i]
```